__tests__/__snapshots__/plugin.test.ts.snap

```diff
@@ -7,7 +7,7 @@ var objX = {
 };
 var _worklet_5359970077727_init_data = {
   code: "function f(){const{x,objX}=this._closure;return{res:x+objX.x};}",
-  location: "${ process.cwd() }/jest tests fixture"
+  location: "/dev/null"
```

Comment:
Good idea, but `/dev/null` sounds a bit magical, let's maybe have just `location: "jest tests fixture"`.

Reply:
Issue with "jest tests fixture" is that in snapshots it gives that absolute directory that has to be manually changed when updating/providing new snapshots. Maybe we should do
```suggestion
  // we are dumping unused babel error output
  location: "/dev/null"
```

Reply:
What about `/jest/tests/fixture`?

Reply:
After our discussion I have changed the comment regarding that line for more direct explanation